src/Google.Pubsub.V1/PublisherClient.cs

```diff
@@ -56,12 +56,79 @@ public sealed partial class PublisherSettings : ServiceSettingsBase
         /// <returns>A new instance of the default PublisherSettings.</returns>
         public static PublisherSettings GetDefault() => new PublisherSettings();
 
+        public static Predicate<RpcException> IdempotentRetryFilter { get; } =
+            RetrySettings.FilterForStatusCodes(StatusCode.Unavailable, StatusCode.DeadlineExceeded);
+
+        public static Predicate<RpcException> NonIdempotentRetryFilter { get; } =
+            RetrySettings.FilterForStatusCodes();
+
+        public static BackoffSettings GetDefaultRetryBackoff() => new BackoffSettings
+            {
+                Delay = TimeSpan.FromMilliseconds(100),
+                DelayMultiplier = 1.2,
+                MaxDelay = TimeSpan.FromMilliseconds(1000),
+            };
+
+        public static BackoffSettings GetDefaultTimeoutBackoff() => new BackoffSettings
+        {
+            Delay = TimeSpan.FromMilliseconds(2000),
+            DelayMultiplier = 1.5,
+            MaxDelay = TimeSpan.FromMilliseconds(30000),
+        };
+
+        public RetrySettings CreateTopic_Retry { get; set; } = new RetrySettings {
```

Comment:
OK, I'll remove the _s. The GetDefault...() methods are only called "Default" because that's how it's defined in the configuration. There may be multiple settings defined, with arbitrary names.